virusseq_metadata_template.tsv

```diff
@@ -1 +1 @@
-specimen collector sample ID	sample collected by	sequence submitted by	sample collection date	sample collection date precision	geo_loc_name (country)	geo_loc_name (state/province/territory)	geo_loc_name (city)	organism	isolate	purpose of sampling	purpose of sampling details	NML submitted specimen type	anatomical material	anatomical part	body product	environmental material	environmental site	collection device	collection method	host (scientific name)	host disease	host age	host age unit	host age bin	host gender	purpose of sequencing	purpose of sequencing details	sequencing date	library ID	sequencing instrument	sequencing protocol name	raw sequence data processing method	dehosting method	consensus sequence software name	consensus sequence software version	breadth of coverage value	depth of coverage value	consensus genome length	Ns per 100 kbp	reference genome accession	bioinformatics protocol	lineage/clade name	lineage/clade analysis software name	lineage/clade analysis software version	variant designation	variant evidence	variant evidence details	study_id
+specimen collector sample ID	sample collected by	sequence submitted by	sample collection date	sample collection date null reason	geo_loc_name (country)	geo_loc_name (state/province/territory)	organism	isolate	fasta header name	purpose of sampling	purpose of sampling details	anatomical material	anatomical part	body product	environmental material	environmental site	collection device	collection method	host (scientific name)	host disease	host age	host age null reason	host age unit	host age bin	host gender	purpose of sequencing	purpose of sequencing details	sequencing instrument	sequencing protocol	raw sequence data processing method	dehosting method	consensus sequence software name	consensus sequence software version	breadth of coverage value	depth of coverage value	reference genome accession	bioinformatics protocol	gene name	diagnostic pcr Ct value	study_id

```

Comment:
there some weird casing here - perhaps it should just all be lwoercase for these two?
- specimen collector sample ID
- diagnostic pcr Ct value

Reply:
@rosibaj these are field names in submission TSV which are synced with DataHarmonizer's template field names. It should not have any impact for our internal usage, as Jaser and Alex already have script to handle these casing without issue. You can see:
https://github.com/cancogen-virus-seq/muse/blob/submission_tsv_adjustment/src/main/resources/tsv-schema.json
https://github.com/cancogen-virus-seq/singularity/blob/6b9dc2c678c4e7893f7f3e5a984551efe135ea7d/src/main/java/org/cancogenvirusseq/singularity/utils/TsvUtils.java#L36-L76